Modules/Module_Import/xASL_wrp_DCM2NII_Subject.m

```diff
@@ -103,6 +103,12 @@
             end
             
             %% 4. Iterate over scans
+			% Safety-check, we can't allow multiple matches for the same sequence
+			if numel(thisRun.scanIDs) ~= numel(unique(thisRun.scanIDs))
+				fprintf('Detected duplicit scans: %s\n',strjoin(thisRun.scanIDs, ', '));
```

Comment:
duplicit -> duplicate

No, “duplicit” is not a correct word in English. The correct word you’re thinking of is “duplicate”, which refers to something that is an exact copy or reproduction of another item.

There might be confusion with “duplicitous,” which is an adjective used to describe someone as being deceitful or two-faced. However, “duplicit” on its own is not recognized as an English word.

Thanks chatGPT :)

Reply:
nice. fixed.